### PR TITLE
updating docs on how to make jump menu command for clarity

### PR DESCRIPTION
**Pull request by TomJaeger**
Updated docs for clarity on building a jump menu command.  Also added documentation for permissions array element.

### CODE REVIEW

**Review by intoeetive — APPROVED** [review]

File: docs/development/jump-menu.md [hunk 1]
```diff
@@ -17,33 +17,50 @@ Add-ons can easily add custom items for the Jump Menu.
...
+        ));
 
-3. Add commands for your custom items.   
-Item commands are comprised of the following:
+To Add Jump Menu commands to your add-on, you simply add array elements to the $items array in the example above
 
-- `[commandTitle]` _(string)_ Unique command title used as key in global jumps array. Will be prefixed with `[addonName]:` so a command title in Assets of `editS3Source` will be `Assets:editS3Source`
+The array of a Jump Menu command is comprised of the following keys:
+
+- `commandArrayTitle` _(string)_ Unique command title used as key in global jumps array. Will be prefixed with the add-ons name so a command title in the add-on Assets of `editS3Source` will be `Assets:editS3Source`
```
Comment (intoeetive) [inline docs/development/jump-menu.md]: I think this should be `commandTitle` and not `commandArrayTitle`?

Also I feel like the text needs some proofreading - tagging @matthewjohns0n

**Review by intoeetive — CHANGES_REQUESTED** [review]

File: docs/development/jump-menu.md (hunk 1 above)
Comment (TomJaeger) [reply docs/development/jump-menu.md]: Theres a second command_title lower as a secondary array element, that has to be kept... Changed it to array title to make it unique, so people would know which one is being referenced.

File: docs/development/jump-menu.md [hunk 2]
```diff
@@ -17,33 +17,50 @@ Add-ons can easily add custom items for the Jump Menu.
 
 NOTE: Sample File: https://gist.github.com/obfuscode/77d0135970c9ce8bc84139c127507bb1
```
Comment (matthewjohns0n) [inline docs/development/jump-menu.md]: We should update this link to point to a gist we have access to.

**Review by matthewjohns0n — CHANGES_REQUESTED** [review]

File: docs/development/jump-menu.md (hunk 2 above)
Comment (TomJaeger) [reply docs/development/jump-menu.md]: Removed this... especially since we have the generator now.

**Review by matthewjohns0n — APPROVED** [review]

**Pull request merged**